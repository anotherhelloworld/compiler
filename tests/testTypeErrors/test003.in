var 
    a : integer = 10;
    b : real = 10 ;
begin
    if a = true then 
        b := a; 
end.