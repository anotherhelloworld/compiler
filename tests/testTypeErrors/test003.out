Incompatible types: expected: "integer" but "boolean" found Line 5 Column 17
